src/Stoa.ts

```diff
@@ -22,7 +22,6 @@ class Stoa {
             if (err != null)
             {
                 console.error(err);
-                throw new Error(err.message);
```

Comment:
Why would the ctor not throw if there's an error ?

Reply:
It's my mistake. I added it again.